### PR TITLE
currency-tailwind-design

### PR DESCRIPTION
**Pull request by hebaali9**
Screenshot 2023-01-15 at 11 10 05 PM

Screenshot 2023-01-15 at 11 10 21 PM

### CODE REVIEW

File: src/currencyPage.tsx [hunk 1]
```diff
@@ -16,56 +15,95 @@ function CurrencyPage() {
...
-      <br></br>
-      <form>
-        <Input
-          className="antd-input"
-          type="number"
-          value={fromAmount.toString()}
-          onChange={handleChange}
-        ></Input>
+      <div className="md:mx-48 text-blue-900 bg-stone-100 shadow-2xl md:flex md:flex-col md:items-center  md:h-80">
+        <p className="font-sans mt-6 text-center md:mt-6 md:mb-16 md:text-xl md:font-bold">
```
Comment (SirajKakeh) [inline src/currencyPage.tsx:46]: we don't need md:mt-6 here since it's already mt-6

File: src/currencyPage.tsx [hunk 2]
```diff
@@ -16,56 +15,95 @@ function CurrencyPage() {
...
 
-        <Select value={fromCurrency} onChange={(e) => setFromCurrency(e)}>
-          {Object.keys(currencies).map((currency) => (
-            <Select.Option key={currency} value={currency}>
-              {currency}
-            </Select.Option>
-          ))}
-        </Select>
+        <form className=" mt-10 flex flex-col md:flex-row md:flex md:items-center md:my-6 ">
+          <h2 className=" ml-6 md:flex md:ml-3 md:mx-5">From</h2>
```
Comment (SirajKakeh) [inline src/currencyPage.tsx:51]: md:mx-5 will overwrite md:ml-3 here, so let's stick to only one of them

**Review by SirajKakeh — CHANGES_REQUESTED** [review]
site looks good, left some comments

**Review by SirajKakeh — CHANGES_REQUESTED** [review]
site looks good, left some comments

File: src/currencyPage.tsx [hunk 3]
```diff
@@ -16,56 +15,95 @@ function CurrencyPage() {
     });
   }, []);
 
+  useEffect(() => {
+    mockCurrencyAPI;
+  }, [fromCurrency, toCurreny]);
```
[the remark this answers is not in the document]
Comment (hebaali9) [reply src/currencyPage.tsx:20]: i was trying to change the amount when change  the currency 
i saw this method in video

[the remark this answers is not in the document]
Comment (SirajKakeh) [reply src/currencyPage.tsx:20]: yeah but you actually need to change the amount inside the use effect, what you have here is just `mockCurrencyAPI` that you're not calling so it's doing nothing, also it shouldn't be here as it's not related to changing the values of the currencies

**Comment by hebaali9** [comment]
can i merg the branch????

**Pull request merged**